docs/references/http-websocket-apis/public-api-methods/server-info-methods/feature.md

```diff
@@ -0,0 +1,165 @@
+---
+html: feature.html
+parent: server-info-methods.html
+seo:
+    description: Get information about protocol amendments.
+labels:
+  - Blockchain
+  - Core Server
+---
+
+# feature
+
+[[Source]](https://github.com/XRPLF/rippled/blob/master/src/ripple/rpc/handlers/Feature1.cpp "Source")<br/>
+
+The `feature` command returns information about [amendments](../../../../concepts/networks-and-servers/amendments.md) this server knows about, including whether they are enabled and if the server knows how to apply the amendments.
+
+This is the non-admin version of the [`feature` admin command](../../admin-api-methods/status-and-debugging-methods/feature.md). It follows the same formatting as the _admin_ command, but hides potentially sensitive data.
+
+{% badge href="https://github.com/XRPLF/rippled/releases/tag/2.2.0" %}New in: rippled 2.2.0{% /badge %}
+
+## Request Format
+
+An example of the request format:
+
+{% tabs %}
+
+{% tab label="WebSocket" %}
+```json
+{
+  "id": "list_all_features",
+  "command": "feature"
+}
+```
+{% /tab %}
+
+{% tab label="JSON-RPC" %}
+```json
```

Comment:
IMO it might be a bit clearer if all the examples are using the same param setup - right now WS has no params and JSON/CLI do